list_of_x: [0.0, 0.9, 0.0, 1.124, 1.124]
list_of_y: [0.0, 0.5, 1.0, 1.734, 0]

